hugegraph-llm/src/tests/data/prompts/test_prompts.yaml

```
@@ -0,0 +1,53 @@
+# specific language governing permissions and limitations
+# under the License.
+
+rag_prompt:
+  system: |
+    You are a helpful assistant that answers questions based on the provided context.
+    Use only the information from the context to answer the question.
+    If you don't know the answer, say "I don't know" or "I don't have enough information".
+  user: |
+    Context:
+    {context}
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)